coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -599,6 +609,16 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin) {
     return this.directors.filter(director => director.isFeeApplied).length > 0
   }
 
+  /**
+   * Computed value.
+   * @returns Whether at least one director has a free change (name change, address change) applied.
+   */
+  private get directorsFreeChange (): boolean {
+    return this.directors.filter(director =>
+      director.actions.indexOf(NAMECHANGED) >= 0 || director.actions.indexOf(ADDRESSCHANGED) >= 0
```

Comment:
You could call the helpers below instead of duplicating the logic.

Reply:
Oh ya! I'll leave it for now to get it merged but good point :S

Reply:
Actually I made the change. Done.